### ISSUE TITLE
deprecation warning on HA Core 2024.10.0b0

### ISSUE THREAD

**Issue by Anto79-ops** [music-assistant/hass-music-assistant#2967]
### What version of Music Assistant has the issue?

2024.9.1

### What version of the Home Assistant Integration have you got installed?

2024.10.0b0

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

hello

Updating to HA Core 2024.10.0b0 and saw these in the logs:

```
Logger: homeassistant.components.media_player.const
Source: helpers/deprecation.py:197
integration: Media player (documentation, issues)
First occurred: 1:20:57 PM (11 occurrences)
Last logged: 1:20:58 PM

MEDIA_TYPE_ALBUM was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.ALBUM instead, please report it to the author of the 'mass' custom integration
MEDIA_TYPE_ARTIST was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.ARTIST instead, please report it to the author of the 'mass' custom integration
MEDIA_TYPE_PLAYLIST was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.PLAYLIST instead, please report it to the author of the 'mass' custom integration
MEDIA_TYPE_TRACK was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.TRACK instead, please report it to the author of the 'mass' custom integration
```

### How to reproduce

update to HA Core 2024.10.0b0

### Music Providers

Radio

### Player Providers

Home Assistant media/cast devices, DNLA

### What version of Home Assistant Core are your running

2024.10.0b0

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by OzGav** [comment]
This will be fixed in the next integration update

**Comment by jonwaland** [comment]
adding the following incase someone is searching...

```
[homeassistant.components.media_player.const] MEDIA_CLASS_ALBUM was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.ALBUM instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_CLASS_ARTIST was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.ARTIST instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_CLASS_DIRECTORY was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.DIRECTORY instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_CLASS_MUSIC was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.MUSIC instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_CLASS_PLAYLIST was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.PLAYLIST instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_CLASS_TRACK was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.TRACK instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_TYPE_ALBUM was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.ALBUM instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_TYPE_ARTIST was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.ARTIST instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_TYPE_PLAYLIST was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.PLAYLIST instead, please report it to the author of the 'mass' custom integration
[homeassistant.components.media_player.const] MEDIA_TYPE_TRACK was used from mass, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaType.TRACK instead, please report it to the author of the 'mass' custom integration
```

**Comment by Anto79-ops** [comment]
Thanks for your comment! Just to confirm, it was _not_ addressed in the latest release 2024.10.0.

Cheers

**Comment by marcelveldt** [comment]
> Thanks for your comment! Just to confirm, it was _not_ addressed in the latest release 2024.10.0.
> 
> Cheers

And it probably will not be or have super low prio - its just a warning about a deprecation in 1 year,
We're in the process of adding the integration to HA core natively, that has higher prio

**Comment by Anto79-ops** [comment]
awesome, thats exciting!

**Comment by OzGav** [comment]
This is fixed in 2024.10.1
